lib/screens/home/home_view_model.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:flutter_riverpod/flutter_riverpod.dart';
+import 'package:lydiaryanfluttersurvey/model/response/surveys_response.dart';
+import 'package:lydiaryanfluttersurvey/model/ui/survey_ui_model.dart';
+import 'package:lydiaryanfluttersurvey/screens/home/home_state.dart';
+import 'package:lydiaryanfluttersurvey/usecases/base/base_use_case.dart';
+import 'package:lydiaryanfluttersurvey/usecases/get_surveys_use_case.dart';
+import 'package:rxdart/subjects.dart';
+
+class HomeViewModel extends StateNotifier<HomeState> {
+  final GetSurveysUseCase _getSurveysUseCase;
+
+  final BehaviorSubject<List<SurveyUiModel>> _surveys = BehaviorSubject();
+  Stream<List<SurveyUiModel>> get surveys => _surveys.stream;
+
+  HomeViewModel(this._getSurveysUseCase) : super(const HomeState.init());
+
+  Future<void> getSurveys() async {
```

Comment:
@ryan-conway should we write unittest for this VM?

Reply:
@manh-t Unit test added in [1a98c09](https://github.com/nimblehq/flutter-ic-lydia-ryan/pull/79/commits/1a98c09797dc6bdaee5ae8d03bf49e6182cff9b4) 🙏🏻